.github/workflows/fmt.yml

```diff
@@ -24,6 +25,8 @@ jobs:
             github.com:443
             objects.githubusercontent.com:443
             pypi.org:443
+            rekor.sigstore.dev
```

Comment:
no port needed here? The rest force 443.

Reply:
2 seconds too late 😂